competition.tex

```diff
@@ -44,56 +41,67 @@ \section{Risks}
-be used by SQEP Blueshirts, and will be safely stored when not in use.}
-{SC - Events}
+{Tools should only be used in appropriate circumstances and in the manner they are designed to be used.
+All use should be supervised by a responsible adult.
+Student Robotics will not provide any tools to competitors.
+First aid provision available to manage any incidents.}
+{Health and Safety Lead, Team Leaders}
 {4}
-
-\risk{}
-{Teams will bring their own tools - Team Leaders must ensure tools are fit for purpose, that students using the tools are competent, and are wholly responsible for their use.}
-{Team Leaders}
-{}
 \hline
 
 \risk{Interaction with robots: electric shock, minor injury}
 {Team Leaders to supervise work on robots in team pits, Blueshirts will also intervene if work seems unsafe.
-Food and Drink is not permitted in team pit areas.}
-{SC - Events, Team Leaders}
+Only sealed food and drink is permitted in team pit areas.}
+{Health and Safety Lead, Team Leaders}
 {3}
+\hline
 
-\risk{}
-{Robots subject to a safety inspection before entry into an arena. Arena access controlled by
-Blueshirts - maximum of 4 teams at a time, and modification of robots inside the arena is banned.}
-{SC - Events}
-{}
-
+\risk{Injury to competitors, Blueshirts, and members of the public due to unsafe robots}
+{Robots subject to a safety inspection before entry into an arena.
+Robots re-inspected randomly throughout the event, entry into the arena can be revoked at any time.
+Anyone identifying a potential safety issue to report it to the safety inspector.
+Arena access controlled by Blueshirts - maximum of 4 teams at a time, and modification of robots inside the arena is banned.
+Power cut off switch to be readily accessible on robots.}
+{Health and Safety Lead}
+{3}
 \hline
 
 \risk{Trip Hazard from trailing extension leads}
 {Extension leads taped down and inspected regularly, kept away from walkways
 where reasonably practicable. Blueshirts and Team Leaders to enforce teams
 keeping within their areas and that areas are kept tidy}
-{SC - Events, Team Leaders}
+{Health and Safety Lead, Team Leaders}
 {1}
 \hline
 
+\risk{Falling on stairs}
+{Carrying of Robots or large or heavy objects on the stairs to be kept to a minimum. Running on the stairs is not permitted.}
+{Health and Safety Lead, Team Leaders}
+{2}
+\hline
+
 \risk{Battery failure - smoke, fire}
-{Teams will hand over batteries and chargers on arrival, from this point
-SQEP Blueshirts to handle battery charging in a designated area. Blueshirts
-and Team Leaders to identify batteries showing signs of damage or swelling
-and deliver to Helpdesk for safe disposal.}
-{SC - Events, Team Leaders}
+{All batteries to be charged by trained volunteers in fire-proof bags.
+Competitors and team leaders have been informed about safe use of the batteries throughout the competition year.
+Blueshirts and Team Leaders to identify batteries showing signs of damage or swelling and deliver to Helpdesk for safe disposal.}
```

Comment:
Hrm, this should impact https://github.com/srobo/tasks/issues/122 (helpdesk procedures) -- we should have a means there for the safe storage/disposal of any bad batteries (see also https://github.com/srobo/tasks/issues/121, battery charging procedure)